nmoscommon/utils.py

```diff
@@ -54,3 +78,82 @@ def getLocalIP():
                 pass
     # Could not find an interface
     return None
+
+
+def downgrade_api_version(
```

Comment:
'downgrade_ver' is very specific to the IS-04 Query API. As such I'd suggest renaming the function and downgrade_map constant/parameter to 'translate_api_version' and 'translation_map' as-per https://amwa-tv.github.io/nmos-discovery-registration/branches/v1.3-dev/docs/6.0._Upgrade_Path.html#version-translations